.docker/ci/release/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:20.04
```

Comment:
22.04 is latest LTS, we should probably use that one.
We use `ubuntu-latest` when building, but that's prob 22.10, so for running I'd propose 22.04 as base.

Reply:
Yes, our dedicated worker runs on 22.04.1. I will update here as well and test. Thanks.

Reply:
> 💻 Linux distribution: Ubuntu 22.04.1 LTS

Bumped and tested docker image to run node with 22.04 version and it runs successfully.